internal/cmd/commands/server/server.go

```diff
@@ -133,21 +134,12 @@ func (c *Command) AutocompleteFlags() complete.Flags {
 }
 
 func (c *Command) Run(args []string) int {
-	ctx := context.TODO()
```

Comment:
Reviewers: In a few of these commands context.TODO had been inserted. This was from a previous change that added updated error into various files that required context. I believe the person that did that work simply did not realize that each command already has a context baked in. I can see no reason to use TODO or Background here.